apps/hashdotai/guide/04_types/04_data-types.mdx

```diff
@@ -19,15 +19,16 @@ HASH contains a series of primitive data types, which are widely used, and from
 - `Number`: an arithmetical value (in the [real number](https://en.wikipedia.org/wiki/Real_number) system)
 - `Boolean`: a "true" or "false" value
 
-HASH also contains three other primitive data types, which are unlikely to be useful to most users.
+HASH also contains a small number of other primitive data types, which are unlikely to be useful to most users.
 
-<Toggle title="The 3 advanced primitive data types">
+<Toggle title="Advanced primitive data types">
 
-The following primtive data types are also supported in HASH, ensuring that all values representable within JavaScript Object Notation (JSON) are also representable in HASH:
+The following primtive data types are also supported in HASH:
 
 - `Null`: A placeholder value representing 'nothing'
 - `Object`: A plain JSON object with no predefined structure
-- `Empty List`: An empty List
+
+Historically, an `Empty List` data type was also supported, ensuring that all possible values representable within JavaScript Object Notation (JSON) were also representable in HASH by default, but this was later removed as a generally-available "primitive".
```

Comment:
```suggestion
Historically, an `Empty List` data type was also provided, ensuring that all possible values representable within JavaScript Object Notation (JSON) were also provided for by primitive data types in HASH (by default). This was later removed, and to replicate its functionality (if required) you should use an array without any elements.
```

Reply:
@TimDiekmann Is this accurate/could it be better explained?

Reply:
I would probably just drop that line TBH

Reply:
Agreed

Reply:
```suggestion
```